test/language/statements/for-of/iterator-next-reference.js

```diff
@@ -3,39 +3,33 @@
 /*---
 es6id: 13.6.4.13 S5.c
 description: >
-    The iterator's `next` method should be accessed with each iteration as per
-    the `IteratorStep` abstract operation (7.4.5).
+    The iterator's `next` method should be accessed only once with each
+    iteration as per the `GetIterator` abstract operation (7.4.1).
 features: [Symbol.iterator]
 ---*/
 
 var iterable = {};
 var iterator = {};
 var firstIterResult = { done: false };
-var iterationCount, invocationCount;
+var iterationCount = 0;
 
 iterable[Symbol.iterator] = function() {
   return iterator;
 };
 
-iterator.next = function() { return { value: 45, done: false }; };
-iterationCount = 0;
-invocationCount = 0;
+iterator.next = function() {
```

Comment:
maybe in a follow up PR - which I can do - or here if you're up to: defining this over a ` Object.defineProperty(iterator, 'next',` get accessor should be better for catching it get's the method only once.

Again, this should not be a blocker.

Reply:
Is this in addition to the defineProperty below, or on its own?

Reply:
the changes you did sounds great already.